rolling/distribution.yaml

```diff
@@ -2023,6 +2023,21 @@ repositories:
       url: https://github.com/ros/resource_retriever.git
       version: ros2
     status: maintained
+  rmf_battery:
+    doc:
+      type: git
+      url: https://github.com/open-rmf/rmf_battery.git
+      version: rolling
```

Comment:
It looks like the `rolling` branch doesn't exist. That said, I would recommend changing this to `main`. Releasing from a distro specific branch like `galactic`, `foxy`, etc is useful for maintaining stability and backwards compatibility, but it comes with some overhead. For ROS Rolling releasing from `main`/`master`/`ros2` is fine because it's expected to be unstable.

Reply:
Sorry the `rolling` branch got deleted after a PR was merged, I just restored it. 

Thanks for the tip regarding the branch, I agree and would move `rolling` to `main` on the next release, for this one I guess I would rather keep it on the `rolling` branch as we internally agreed before, and for consistency with the other packages.